api/db/populate-friendships.js

```diff
@@ -1,19 +1,42 @@
-const { getRandomSubset, getRandomDate, createPrompt } = require('./utils');
+const { getRandomSubset, getRandomDate } = require('./utils');
+const {
+  createNotification,
+  notificationTypes,
+} = require('./populate-notifications');
 
 const User = require('../models/User');
 const Friendship = require('../models/Friendship');
 
 let displayLogs;
 
 const createFriendship = async (friendshipParams) => {
-  const friendship = new Friendship({
-    requestant: friendshipParams.requestant,
-    receivant: friendshipParams.receivant,
-    accepted: friendshipParams.accepted,
-    acceptedAt: friendshipParams.acceptedAt,
-  });
+  const friendship = new Friendship(friendshipParams);
   await friendship.save();
 
+  // New friendship request notifications
+  await createNotification(
+    {
+      user: friendship.receivant,
+      type: notificationTypes[6],
+      read: friendship.accepted,
+      referencedUser: friendship.requestant,
+      referencedFriendship: friendship,
+    },
+    displayLogs
+  );
+  if (friendship.accepted) {
```

Comment:
I would add a comment before this line identifying the type of the notification too, as you did for all others.